### DISCUSSION TITLE
Activate Github Sponsors

### DISCUSSION THREAD

**Discussion by thibaultmol** [cpriest/SnapLinksPlus#333] [General]
https://github.com/sponsors

(doesn't have to be github sponsors, can be 3rd party donation systems like Patreon, Librepay etc.
But don't see a reason not to enable the actual github sponsors system)
https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository

**Comment by cpriest** [comment]
I'm not sure what the aim would be.  The project its-self just needs time & people, donation of money (to me) is something I did in the past via Mozilla and it made me feel obligated to work on it which I didn't like, in fact, it killed my desire to work on the project.

Setting it up for donations to a charity might be a good setup though, what are your thoughts?

Fortunately, I am not wanting for money, but time.

**Comment by thibaultmol** [comment]
I think if the donations are going to a charity, people will just donate to those charities directly.

Yeah, I feel you. My limit is also time not money. I wish that Github had a builtin system for bug bounty's. 
Bountysource.com is the main tool that I've found for this, but it's annoying because there is nothing in the github interface that makes it apparent that something has a bounty on it
